model/Core/Vocabularies/RelationshipType.md

```diff
@@ -12,73 +12,79 @@ Provides information about the relationship between two Elements.
-- buildDependency: Every `to` Element is a build dependency of the `from` Element
-- buildTool: Build tool used to build an Element. This may be used to describe the build tool of a Build instance
+- affects: (Security/VEX) The `from` vulnerability affect each `to` Element
+- amendedBy: The `from` Element is amended by each `to` Element
+- hasAncestor: The `from` Element has ancestor described by each `to` Element
+- availableFrom: The `from` Element is available from the additional supplier described by each `to` Element
+- hasBuildDependencyOn: The `from` element has a build dependency on each `to` Element
+- usesBuildTool: The `from` Build element uses each `to` element as a build tool
 - coordinatedBy: (Security) Used to identify the vendor, researcher, or consumer agent performing coordination for a vulnerability
-- concludedLicense: The `to` specifices the license the SPDX data creator has concluded as governing the `from` software Artifact.
-- contains: Every `to` Element is contained by the `from` Element
-- configOf: (Build) Configuration information applied to an Element instance during a LifeycleScopeType period.  Example: Build configuration of the build instance
-- copy: Every `to` Element is a copy of the `from` Element
-- dataFile: Every `to` Element is a data file related to the the `from` Element
-- declaredLicense: The `to` license information identifies the license information actually found in the `from` software Artifact, for example as detected by use of automated tooling.
-- dependencyManifest: Every `to` Element is manifest file containing dependency information related to the `from` Element
-- dependsOn: Every `to` Element is a dependecy of the `from` Element
-- descendant: This relationship may be used to describe child builds of a Build instance.
-- describes: Every `to` Element is  described by the `from` Element.  This can be used to denote the root(s) of a tree of elements contained in an SBOM.
-- devDependency: Every `to` Element is a development dependency for the `from` Element
-- devTool: Every `to` Element is a development tool for the `from` Element
-- distributionArtifact: Every `to` Element is an artifact intended for distribution of the `from` Element (e.g. an RPM or archive file)
-- documentation: Every `to` Element is documentation for the `from` Element
-- doesNotAffect: (Security/VEX) Specifies a vulnerability has no impact on one or more elements
-- dynamicLink: Every `to` Element is dynamically linked to the `from` Element
-- example: Every `to` Element is an example for the `from` Element
-- evidenceFor: (Dataset) Every `to` Element is can be considered as evidence for the `from` Element
-- expandedFromArchive: Every `to` Element is an artifact expanded from the `from` archive file
+- hasConcludedLicense: The `from` software Artifact is concluded by the SPDX data creator to be governed by each `to` license
```

Comment:
not sure we want to change from concludedLicense to hasConcludedLicense?

s/software Artifact/Artifact/ will future proof this.  Otherwise  software should be capitalized.

Reply:
The intention is to preserve the original order of the relationship, but make it clear from the name what the directionality is (e.g. I'm not changing how the relationship is used, just the name). `concludedLicense` is ambiguous as to the direction, and TBH I'd rather have good names that are easy for new users to understand than worry about exactly maintaining the names from SPDX 2